src/usr/local/www/pkg_edit.php

```diff
@@ -529,12 +515,17 @@ function parse_package_templates() {
 	}
 }
 
-// Start of page display
-if ($input_errors)
-	print_input_errors($input_errors);
+//breadcrumb
+if ($pkg['title'] != "") {
+	if(!$only_edit) {
+		$pgtitle = array($pkg['title'], gettext("Edit"));
+	} else {
+		$pgtitle = array($pkg['title']);
+	}
+} else {
+	$pgtitle = array(gettext("Package Editor"));
+}
```

Comment:
Tracing head.inc and the functions it uses shows that the $pgtitle array must have at least two elements if it is to be printed on the page, so I have committed this PR  but with:

$pgtitle = array(gettext("Package"), $pkg['title']);

Need to check if the >= 2 elements thing is still valid, but this works for now.